### DISCUSSION TITLE
opensensemap.org integration

### DISCUSSION THREAD

**Discussion by schematt** [Daveiano/weewx-wdc#125] [Ideas]
do you think it is possible to integrate the values of pm 2.5 and pm 10 from opensensemap.org?

**Comment by Daveiano** [comment]
I have the feeling that this should be handled by a separate WeeWX extension, not the skin itself. 

I found this one https://github.com/sbsrouteur/weewx-opensensemap, but it only sends data ### to opensensemap, it does not **get** data from it.

Anyway, their API seems pretty straightforward, do you know about any other integrations?

**Reply by schematt** [reply]
this is hardware inside open sense e https://sensor.community
https://github.com/matthewwall/weewx-sds011

**Reply by schematt** [reply]
if i understand correctly using the interceptor driver i should have two weewx running and creating two separate databases

A WeeWX process has at most one "driver" to communicate with the station hardware and receive "high resolution" (i.e. every few seconds) measurement data in the form of LOOP packets. The driver is single-threaded and blocking, so no more than one driver can run in a WeeWX process.

**Reply by schematt** [reply]
Maybe this discussion can help you

https://groups.google.com/g/weewx-user/c/UJhz34G5r8s/m/eT0KCy7sBAAJ

**Reply by Daveiano** [reply]
Sorry, you wrote _integrate from opensensemap.org_. So I thought you want to connect to a station via the website!?

But you have a hardware station by yourself and want to connect to that station, right? You only need two weewx instances if you want to use multiple drivers, this is correct. If so (if you need to run multiple drivers), I found a discussion about that in the weewx user group: https://groups.google.com/g/weewx-user/c/zL6dnD7el6k
